README.md

```diff
@@ -29,16 +29,29 @@ main() async {
 }
 ```
 
-### Create and monitor a list of services
+### Create and monitor a list of Reachables
```

Comment:
"list of reachable services" maybe?

Reachable won't make any sense to folks seeing this for the first time.